guides/doc-Administering_Red_Hat_Satellite/topics/proc_integrating-satellite-with-red-hat-single-sign-on-for-external-authentication.adoc

```diff
@@ -82,39 +71,49 @@ For more information about audience support, see https://access.redhat.com/docum
 +
 * From the *Mapper Type* list, select *Group Membership*.
 +
-* From the *Token Claim Name* list, select *groups*.
+* In *Token Claim Name*, enter *groups*.
 +
 * Set the *Full group path* toggle to *OFF*.
 +
 For more information about group mappers, see https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.3/html/server_administration_guide/user-storage-federation#ldap_mappers[Group Mapper] in the _LDAP Mappers_ section of the _{Keycloak} Server Administration Guide_.
 
 .. Click *Save*.
-
-. In the {Project} web UI, navigate to *Administer* > *Settings*, and click the *Authentication* tab.
+.. Navigate to *Users* and click *Add user*. 
```

Comment:
I think, it would be better if we created the group before the user, so that user can add itself into a group directly. Else, here the admin is creating a user, then creating a group and then again going to the user to add it into a group. Does it make sense? Let me know your thoughts.

Reply:
Makes sense. Swapped the steps to create group and user. Does it look right?